testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```diff
@@ -0,0 +1,97 @@
+steps:
+- id: 'Generate UUID'
+  name: 'ubuntu'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 15 | head -n 1 > _uuid
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  entrypoint: /bin/bash
+  dir: '${_SAMPLE_DIR}'
+  args:
+  - -c
+  - |
+    docker build -t gcr.io/${PROJECT_ID}/${_SERVICE}:$(/bin/cat _uuid) .
```

Comment:
My thought was to keep the container tag associated with the git hash because we can always look up the container image associated with the service, but it can be confusing to reverse from the container to the code.

We could run the build with `--label` to attach either the SHORT_SHA or this id to the container image, that will show up in gcr.io or on inspections of the container. Either way, we should add a comment explaining how we're tagging/labeling the container.

Reply:
I agree. I see that error in my mental model of the system. Thanks

Reply:
I think it might be overkill to add the label but I've added a comment for the tag